pkg/segment/structs/evaluationstructs.go

```diff
@@ -1519,9 +1519,72 @@ func handleTrimFunctions(op string, value string, trim_chars string) string {
 	}
 }
 
+func formatTime(t time.Time, format string) string {
+	preReplacements := map[string]string{
+		"%e": "_2",
+		"%a": "Mon",
+		"%A": "Monday",
+		"%d": "02",
+		"%b": "Jan",
+		"%B": "January",
```

Comment:
I got them from the time pkg documentation https://pkg.go.dev/time#pkg-constants and also used [strftime.net](https://strftime.net/) to verify

Reply:
Ok can you add a comment indicating that so that later if we have to change/add something we'll have a better idea of how we should set the new value? (But hopefully we'll never have to update this map)